final_task/setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup, find_packages
+import os
+
+setup(name='Rss reader',
+      version='1.4.3',
+      description='Pure Python command-line RSS reader.',
+      author='Marina Romanchuk',
+      author_email='marina-romanchuk-2015@mail.ru',
+      data_files=[('rss_reader', ['rss_reader/ARIALUNI.ttf'])],
```

Comment:
Do you actually need this here? You have this file specified in your `MANIFEST.in`